src/plugins/intel_cpu/src/emitters/plugin/aarch64/jit_eltwise_emitters.cpp

```diff
@@ -261,6 +261,91 @@ void jit_equal_emitter::register_table_entries() {
     push_arg_entry_of("one", 0x3f800000, true);
 }
 
+/// ELU ///
+jit_elu_emitter::jit_elu_emitter(dnnl::impl::cpu::aarch64::jit_generator* host,
+                                 dnnl::impl::cpu::aarch64::cpu_isa_t host_isa,
+                                 const std::shared_ptr<ov::Node>& node)
+        : jit_emitter(host, host_isa, node, get_arithmetic_binary_exec_precision(node)) {
```

Comment:
```suggestion
        : jit_emitter(host, host_isa, get_arithmetic_binary_exec_precision(node)) {
```
In fact, I'm not against on passing alpha. My concern is on the base class jit_emitter. As far as I see, we don't need to pass node to the constructor of jit_emitter, because it's not used anyway. We can use the same base class constructor as in line 282. And we can delete the 2nd constructor of class jit_emitter. What do you think?

Reply:
> In fact, I'm not against on passing alpha. My concern is on the base class jit_emitter. As far as I see, we don't need to pass node to the constructor of jit_emitter, because it's not used anyway. We can use the same base class constructor as in line 282. And we can delete the 2nd constructor of class jit_emitter. What do you think?

Updated, thanks. CVS-140857 to update for other emitters.